radixdlt/src/integration/java/com/radixdlt/consensus/PerfectNetworkTest.java

```diff
@@ -83,4 +81,37 @@ public void given_4_correct_bfts__then_all_should_get_same_commits_consecutive_v
 			.take(time, timeUnit)
 			.blockingSubscribe();
 	}
+
+	@Test
+	public void given_a_correct_bft__then_new_memory_used_should_be_less_than_10MB_over_2_minutes() {
```

Comment:
I don't know what this is really testing for?  It seems like we have guessed at some values that we think are reasonable, but I don't really see why these numbers make any sense.  This is likely to make this test flakey in the long run, and the lack of explanation here for what we are trying to achieve only exacerbates that problem.

Essentially, this test is saying that 5mb/min memory consumption is OK, but clearly this is not actually OK as a steady state, as this would result in approximately 7Gb/day memory consumption.

Better would be a test that said (something like):
1. There should be a startup period, during which time it is OK for memory usage to grow.
2. After this startup period, memory growth should be, on average, zero.

Reply:
All true, this was more of sanity checking where this test used to fail before this PR and now is not. Will implement the recommendation above.